### PR TITLE
Comment component

### PR DESCRIPTION
**Pull request by felire**
### Summary

Create commentBox and integrate with the component of the previous PR (Detail Box).

### Gif

transition3

### CODE REVIEW

File: WBooks/src/app/screens/BookDetail/index.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+import { View, ScrollView } from 'react-native';
 
+import CommentBox from './components/CommentBox';
 import DetailBox from './components/DetailBox';
+import styles from './styles';
 
 function BookDetail() {
-  return <DetailBox />;
+  return (
+    <View style={styles.bookDetailContainer}>
```
Comment (santinucera) [inline WBooks/src/app/screens/BookDetail/index.js]: I will remove this view and apply the styles to the scrollview

File: WBooks/src/app/screens/BookDetail/components/CommentBox/components/Comment/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import styles from './styles';
+
+function Comment({ image, user, comment }) {
+  return (
+    <View style={styles.commentContainer}>
+      <View style={styles.imageContainer}>
+        <Image style={styles.image} source={{ uri: image }} />
+      </View>
+      <View style={styles.infoContainer}>
+        <View>
```
[the remark this answers is not in the document]
Comment (felire) [reply WBooks/src/app/screens/BookDetail/components/CommentBox/components/Comment/index.js]: infoContainer View has two more views, it has the userName and the comment. It's not an empty View. Its functions is the flexDirection column for the two children, and each children has some information

File: WBooks/src/app/screens/Library/components/BookList/index.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ class BookList extends Component {
 
   render() {
     return (
-      <View style={styles.bookContainer}>
+      <View>
```
[the remark this answers is not in the document]
Comment (felire) [reply WBooks/src/app/screens/Library/components/BookList/index.js]: Ok. Do i use fragment when a component or a function return something and the container is unuseful?

File: WBooks/src/app/screens/BookDetail/components/CommentBox/components/Comment/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (santinucera) [reply WBooks/src/app/screens/BookDetail/components/CommentBox/components/Comment/index.js]: Ok, i will rather use a fragment in that case

File: WBooks/src/app/screens/Library/components/BookList/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (santinucera) [reply WBooks/src/app/screens/Library/components/BookList/index.js]: It's something like that, but I will explain it better in the office

**Pull request merged**